tests/playwright/assets/wp-env-cli.ts

```diff
@@ -1,25 +1,6 @@
-import { execSync } from 'child_process';
-import github from '@actions/core';
+export const wpEnvCli = async ( command: string ) => {
+	const port = ( 1 === Number( process.env.TEST_PARALLEL_INDEX ) ) ? '8889' : '8888';
 
-export const wpEnvCli = ( command: string ) => {
-	const cliCommand = ( 1 === Number( process.env.TEST_PARALLEL_INDEX ) ) ? 'tests-cli' : 'cli';
-	const cli = `npx wp-env run ${ cliCommand } `;
-	const logs = execSync( `${ cli }${ command }`, { encoding: 'utf-8' } );
-	// Print log for debuggability
-	// eslint-disable-next-line no-console
-	console.log( logs );
-
-	if ( ! process.env.CI ) {
-		return logs;
-	}
-
-	if ( logs.includes( 'Warning' ) ) {
-		github.warning( logs );
-	}
-
-	if ( logs.includes( 'Error' ) ) {
-		github.error( logs );
-	}
-
-	return logs;
+	const { cli } = await import( '@elementor/wp-lite-env' );
+	await cli( port, command );
```

Comment:
is this function in this PR or is it from somewhere else?

Reply:
I don't understand the question